### ISSUE TITLE
Error in S4HANA Cloud ??

### ISSUE THREAD

**Issue by danchen99** [SAP-samples/abap-platform-refscen-flight#24]
Hello ABAP Reference Scenario Team:

I've just pulled the latest version from the [ABAP-platform-cloud](https://github.com/SAP-samples/abap-platform-refscen-flight/tree/ABAP-platform-cloud) branch to our S4HANA Cloud (2308) environment

I get these warning and active error.

Is this a issue?  

Best Regards

Thank you so much.

![1](https://github.com/SAP-samples/abap-platform-refscen-flight/assets/16522667/884e3809-55c2-48ed-8e51-a4318946562a)

![2](https://github.com/SAP-samples/abap-platform-refscen-flight/assets/16522667/2068df10-2dc7-4f8f-9421-a09e7bbdf5d6)

![3](https://github.com/SAP-samples/abap-platform-refscen-flight/assets/16522667/7f2fc4aa-5493-4955-a48c-fd3ef06c0376)

**Comment by aazipser** [comment]
Hi,

I have also tried to install it, there are some errors that occur and some objects that are missing. I have created the number range objects manually, but I am currently facing a problem:

In the class /DMO/CL_FLIGHT_DATA_GENERATOR it dumps when calling the badi /dmo/data_generation_badi.

These are the errors that occur for me.

Thank you and best regards
Adalbert

![image](https://github.com/SAP-samples/abap-platform-refscen-flight/assets/153853256/121b5aaa-a714-4b95-b02b-86d5b7acf047)
